.ci/Jenkinsfile

```diff
@@ -47,10 +47,11 @@ pipeline {
       }
       post {
         always {
-          archiveArtifacts(allowEmptyArchive: true, artifacts: '.elastic-package/tmp/test/reports/*.xml')
+          sh(label: 'List test results', script: 'pwd && ls -l && ls -l $(pwd)/src/github.com/elastic/elastic-package/build/test-results/')
+          archiveArtifacts(allowEmptyArchive: true, artifacts: '$(pwd)/src/github.com/elastic/elastic-package/build/test-results/*.xml')
```

Comment:
Did you try with `dir("${BASE_DIR}") { archiveArtifacts( artificats: 'relative' ) }`?

Reply:
I will try this, thanks!

Reply:
Looks like my last experiment worked. Turns out I just had a typo in my relative path (`build/build` instead of just `build`). So I'm going with it.